app/views/layouts/_footer.html.erb

```diff
@@ -6,8 +6,7 @@
         <h2 class="govuk-visually-hidden"><%= t('layouts.application.footer.support') %></h2>
         <ul class="govuk-footer__inline-list">
           <li class="govuk-footer__inline-list-item">
-            <a class="govuk-footer__link" href="mailto:apply@digital.justice.gov.uk">
-              <!-- TO DO add in correct link/email -->
+            <a class="govuk-footer__link" href="/contact">
```

Comment:
Using a route helper method and `link_to` would be preferable especially as paths starting with a forward slash can be problematic if the site gets hosted in a sub-domain in the future.
```erb
<%= link_to t('layouts.application.footer.contact'), contact_path, class: 'govuk-footer__link' %>
```
I think `contact_path` will be the correct route helper method.